src/lib/SX127xDriver/SX127x.cpp

```diff
@@ -193,11 +206,28 @@ void SX127xDriver::SetOutputPower(uint8_t Power)
 void ICACHE_RAM_ATTR SX127xDriver::CommitOutputPower()
 {
   if (pwrPending == PWRPENDING_NONE)
+  {
     return;
-
-  pwrCurrent = pwrPending;
-  pwrPending = PWRPENDING_NONE;
-  hal.writeRegister(SX127X_REG_PA_CONFIG, pwrCurrent);
+  }
+  if (SX127X_PA_20DBM == true)
+  {//20dbm
+    pwrCurrent = pwrPending;
+    pwrPending = PWRPENDING_NONE;
+    //20dbm
+    hal.writeRegister(SX127X_REG_PA_DAC, 0x87);  //turn on 3rd amplifier, see 5.4.3
```

Comment:
This register has a restriction of 1% duty cycle which ELRS goes well beyond.

![image](https://user-images.githubusercontent.com/14170229/218877584-708aa82b-4865-439a-8207-fc548d102b44.png)